### DISCUSSION TITLE
Select -> searchable Bug

### DISCUSSION THREAD

**Discussion by ismaeltj9** [filamentphp/filament#14460] [Help] [answered]
### Package

Form builder

### Package Version

v3.2.115

### How can we help you?

Hello all,

When I use relationship with searchable it brings exactly results, but when I use option with searchable it brings more results that should, is it normal?

```php
Select::make('meli_categoria_id')
	->relationship('meliCategoria', 'path_completo')
	->searchable(),
```
![image](https://github.com/user-attachments/assets/14fa8232-8a3f-42fb-805e-dd829b9021f9)

```php
Select::make('meli_categoria_id')
	->options(Categoria::query()->pluck('path_completo', 'id'))
	->searchable(),
```
![image](https://github.com/user-attachments/assets/56cecadb-4d6f-4039-8ebc-e316ed7c5e46)

**Comment by leandrocfe** [comment, marked as answer]
Use [getSearchResultsUsing()](https://filamentphp.com/docs/3.x/forms/fields/select/#returning-custom-search-results)

**Comment by ismaeltj9** [comment]
Thank you @leandrocfe !!
